packages/gatsby-theme-store/i18n/pt.json

```diff
@@ -118,5 +118,10 @@
   "login.button.greeting": "Ola {name}",
   "login.button.action": "Entrar",
 
-  "preview.not-found": "Pre-visualização não econtrada. Esperando por entradas"
+  "preview.not-found": "Pre-visualização não econtrada. Esperando por entradas",
```

Comment:
Just a typing check, can we change "econtrada" to "encontrada"?

Reply:
Nice ! I think we can remove this translation altogether. This was when we still had preview from localstorage.